.github/workflows/scripts/create_fleets.py

```diff
@@ -0,0 +1,121 @@
+"""
+This script creates:
+1. Github repositories for backing openfleets.
+2. Openfleets and commercial fleets based on a consistent format.
+
+Preconditions:
+Before executing this script, githbub cli "gh" and balena cli "balena"
+should be installed and logged in.
+
+Some things will have to be done manually due to lack of support from gh
+or balena cli.
+Manual steps required after running this script:
+1. Link the fleet to its github repo.
+2. Make the fleet public which effectively makes it openfleet. This step can't be
+done before step 1.
+3. Link the fleet url in github description. gh cli provides a homepage option while
+creating repos but not while you are creating it from a template.
+"""
+
+import traceback
+import subprocess
+import hm_pyhelper.hardware_definitions as hdf
+
+
+# openfleet will only have mainnet fleets.
+FLEET_TYPE_NETWORK_MAP = {'commercial': ['mainnet', 'testnet'], 'openfleet': ['mainnet']}
+COMMERCIAL_FLEET_ENVIRONMENT = {
+    'SENTRY_CONFIG': 'https://3234e2c67fe04c3e9f8b0c9b118c415a@o571444.ingest.sentry.io/5725518',
```

Comment:
OVERRIDE_CONFIG_URL is part of only testnet. Will have to be added manually as it uses .rockpi versions of file. That information is not available at this stage. 
We should make that file independent of SBC as it assumes sbc knowledge for i2c and there is no reason to override i2c value in that file. Probably a historical artifact at this stage.